lib/instrumentation/trace-context.js

```diff
@@ -102,18 +102,17 @@ function start (sampled = false) {
   return new TraceContext(buffer)
 }
 
-const bufferSymbol = Symbol('trace-context-buffer')
-
 class TraceContext {
   constructor (buffer) {
-    this[bufferSymbol] = buffer
+    this.buffer = buffer
```

Comment:
If you need to get at the buffer, should we maybe do a getter instead which just returns a copy of the buffer? That way we can continue to prevent tampering.

Reply:
Rolled it back